doc/supported-hardware.md

```diff
@@ -29,6 +29,7 @@ The following SR-IOV capable hardware is supported with sriov-network-operator:
 | Marvell OCTEON Fusion CNF95XX | 177d | b600 |
 | Marvell OCTEON 10 CN10XXX | 177d | b900 |
 | Marvell OCTEON Fusion CNF105XX | 177d | ba00 |
+| Broadcom BCM57504 | 14e4 | 1751 |
```

Comment:
I don't see the other broadcom NICs here. I think we want to keep this merge conflict free from upstream.

Reply:
Meaning we should not list the Broadcom nics here as supported Nics?

Reply:
I think so.

Reply:
Ok, I removed the BCM57504 from the docs to stay consistent with the other Broadcom nics ds